src/qibolab/_core/pulses/pulse.py

```diff
@@ -47,11 +47,7 @@ class Pulse(_PulseLike):
     Pulse amplitudes are normalised between -1 and 1.
     """
     envelope: Envelope
-    """The pulse envelope shape.
-
-    See
-    :class:`qibolab.pulses.envelope.Envelopes` for list of available shapes.
```

Comment:
I wonder whether we should make this public, and reintroduce this comment (here or somewhere else).

It is true that it is annotated with `Envelope`, but that's the base class, while the other one is the enumeration...

However, this is definitely not for 0.2.0

Reply:
Actually, I just realized that `Envelope` is actually public, so I reintroduced this comment in 49133dcf2eb3a0be78e4ae0f8fa70436216b724a. There is a `BaseEnvelope` which is the base class and an `Envelope` type that could also serve as enumeration and is the relevant one here.